lib/src/kinde_flutter_sdk.dart

```diff
@@ -297,13 +297,13 @@ class KindeFlutterSDK with TokenUtils, HandleNetworkMixin {
     }
   }
 
-  Uri _buildEndSessionUrl() {
-    var uri = Uri.parse(_serviceConfiguration.endSessionEndpoint!)
+  WebUri _buildEndSessionUrl() {
+    var uri = WebUri(_serviceConfiguration.endSessionEndpoint!)
```

Comment:
force unwrap looks unsafe. consider,
  if (_serviceConfiguration?.endSessionEndpoint == null || _config == null) {
    throw Exception("Service configuration or config is null");
  }